tests/test_tabulate.py

```diff
@@ -19,17 +19,17 @@
 from create_system import system_Chiral,ChiralModel
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
 def get_component_list():
     def _inner(quantity):
         if quantity in ["E"]:
-            return [""]
+            return [None]
         if quantity in ["berry","V","morb"]:
-            return ["-"+a for a in "z"]
+            return [a for a in "z"]
         if quantity in ["Der_berry","Der_morb"]:
-            return  ["-"+a+b for a in "xyz" for b in "xyz"]
+            return  [a+b for a in "xyz" for b in "xyz"]
         if quantity == "omega2" :
-    	    return ["-zz"]
+    	    return ["zz"]
         raise ValueError(f"unknown quantity {quantity}")
     return _inner
```

Comment:
This can be a simple function, not a fixture. (Then, we do not need `_inner`)

A fixture-function is needed if one needs to test something inside it, such as in the comparers.

Reply:
Also, why do you compare only z component for Berry, V  morb?

Reply:
Changed to function, changed "z" yo "xyz"